pandas/tseries/frequencies.py

```diff
@@ -104,6 +104,8 @@ def get_period_alias(offset_str: str) -> str | None:
     """
     Alias to closest period strings BQ->Q etc.
     """
+    if offset_str == "ME":
+        return "M"
```

Comment:
`_offset_to_period_map ` already maps `'ME'` to `'M'`, is this still necessary?

Reply:
I agree, it isn’t necessary. I removed it.